src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -271,6 +271,9 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
   }
 
   public isCurrentlyDisplayed() {
+    if (!this.isPristine()) {
```

Comment:
I would invert the condition from negative to positive. Could also use a ternary. e.g.
```
return this.isPristine() ? this.hasValues : true;
```